filetree/CMakeLists.txt

```diff
@@ -1,16 +1,16 @@
 #
 # Copyright (c) 2011-2012 EditorConfig Team
 # All rights reserved.
-# 
```

Comment:
Please make these trailing whitespace removal into a separate commit.

Reply:
Sorry, I just noticed you have an `.editorconfig` file saying `trim_trailing_whitespace = false`. Hence I added that whitespace back in 861284f